C-Plus-Plus/cp/Spiral_Matrix.cpp

```diff
@@ -0,0 +1,102 @@
+class Solution {
+public:
+    vector<int> spiralOrder(vector<vector<int>>& matrix) {
+        vector<int> arrLL;
+        
+        //EDGE CASE
+        if (matrix.size()  == 0) return arrLL;
+        
+        int m = matrix.size() ;
```

Comment:
Instead of variable names like m and n use a good name like `matrix_size`

Reply:
Changing it to rows and columns cause if I write matrix_size, so it would be difficult to distinguish between rows and columns.